src/main/java/util/CreatePet.java

```diff
@@ -0,0 +1,34 @@
+package util;
+
+import io.restassured.http.ContentType;
+import models.Pet;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+public class CreatePet {
+    String  BASE_URL = "https://petstore.swagger.io/v2/pet/";
+
+    public void createPet(Pet newPet) {
+        given().
+                contentType(ContentType.JSON).
+                body(newPet).
+        when().
+                post(BASE_URL).
+        then().
+                log().body().
+        assertThat().
+                contentType("application/json").
+                statusCode(200).
+                body("id", equalTo((int) newPet.getId())).
```

Comment:
these assertions could be simplified if you compared the deserialized response to the input. For this you have to teach your classes how to compare them. If it's not working by default, you can override the `equalTo()` method.